src/utils/common.ts

```diff
@@ -89,14 +90,10 @@ export const getAddressUrl = (chainId: number, address: string) => {
   const network = getNetworkById(chainId)
   const explorerUrl = network.data.params[0].blockExplorerUrls[0]
 
-  if (network.chain_id > 0) {
+  if (network.chain_id > 0 || [NETWORK_IDS.Solana, ...COSMOS_CHAINS].includes(network.chain_id as any)) {
```

Comment:
DON't commit, only suggestion (not sure about utils naming)

```suggestion
  if ([isEvmChain(network.chain_id), isSolChain(network.chain_id), isCosmosChain(network.chain_id)].some(el => el === true)) {
```

Reply:
Not all solana chains are available, because `SolanaTestnet` should refer to the `return `${explorerUrl}/account/${address}?cluster=testnet` pattern